security/cmd/flexvolume/driver/driver.go

```diff
@@ -78,15 +83,6 @@ type ConfigurationOptions struct {
 	// Default: /creds
 	// For example: /creds here implies /var/run/nodeagent/creds/ directory
 	NodeAgentCredentialsHomeDir string `json:"nodeagent_credentials_home,omitempty"`
-	// Whether node agent will be notified of workload using gRPC or by creating files in
-	// NodeAgentCredentialsHomeDir.
-	// Default: false
-	UseGrpc bool `json:"use_grpc,omitempty"`
```

Comment:
Can we keep this option? The backward compatibility will make co-workers' life easier since we are familiar with it and we are likely to keep it eventually.